<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APProject</class>
 <widget class="QMainWindow" name="APProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>801</width>
    <height>551</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AP Word</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16777215</width>
      <height>16777215</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>801</width>
      <height>551</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="tabStopWidth">
     <number>75</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_As_PDF"/>
    <addaction name="actionClose_2"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionundo"/>
    <addaction name="actionredo"/>
    <addaction name="actionCopy"/>
    <addaction name="actioncut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionClear"/>
    <addaction name="actioninsert_image"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actioncolor"/>
    <addaction name="actionfont"/>
    <addaction name="actionMinimize"/>
    <addaction name="actionNormalScreen"/>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuhighlight">
    <property name="title">
     <string>Highlight</string>
    </property>
    <addaction name="actionpython"/>
    <addaction name="actionc"/>
    <addaction name="actiondefault"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuhighlight"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>Resources/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-new-folder-48.png</normaloff>Resources/icons8-new-folder-48.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-close-48.png</normaloff>Resources/icons8-close-48.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-save.gif</normaloff>Resources/icons8-save.gif</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As_PDF">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-pdf-48.png</normaloff>Resources/icons8-pdf-48.png</iconset>
   </property>
   <property name="text">
    <string>Save As PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-close-48.png</normaloff>Resources/icons8-close-48.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-copy-file-48.png</normaloff>Resources/icons8-copy-file-48.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-paste-48.png</normaloff>Resources/icons8-paste-48.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-clear-64.png</normaloff>Resources/icons8-clear-64.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-save-as-48.png</normaloff>Resources/icons8-save-as-48.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-color-48.png</normaloff>Resources/icons8-color-48.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionfont">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-font-48.png</normaloff>Resources/icons8-font-48.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-minimize-64.png</normaloff>Resources/icons8-minimize-64.png</iconset>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionNormalScreen">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-normal-screen-64.png</normaloff>Resources/icons8-normal-screen-64.png</iconset>
   </property>
   <property name="text">
    <string>Normal Screen</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-full-screen-64.png</normaloff>Resources/icons8-full-screen-64.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionpython">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-python-48.png</normaloff>Resources/icons8-python-48.png</iconset>
   </property>
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionc">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-c-48.png</normaloff>Resources/icons8-c-48.png</iconset>
   </property>
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-cut-48.png</normaloff>Resources/icons8-cut-48.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-undo-48.png</normaloff>Resources/icons8-undo-48.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-redo-48.png</normaloff>Resources/icons8-redo-48.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actioninsert_image">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-insert-raster-image-48.png</normaloff>Resources/icons8-insert-raster-image-48.png</iconset>
   </property>
   <property name="text">
    <string>Insert image</string>
   </property>
  </action>
  <action name="actiondefault">
   <property name="icon">
    <iconset>
     <normaloff>Resources/icons8-default-64.png</normaloff>Resources/icons8-default-64.png</iconset>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
